doc/whats_new/v1.2.rst

```diff
@@ -44,6 +44,12 @@ Changelog
 - |Enhancement| :class:`cluster.Birch` now preserves dtype for `numpy.float32`
   inputs. :pr:`22968` by `Meekail Zain <micky774>`.
 
+:mod:`sklearn.decomposition`
+............................
+
+- |Fix| Avoid spurious warning in :class:`decomposition.IncrementalPCA` when
+  `n_samples == n_components`:pr:`23264` :user:`Lucy Liu <lucyleeow>`
```

Comment:
I think the what's new should target 1.1.1 (in v1.1.rst) since this is a bugfix.

Reply:
Whoops sorry, I see how it works now :sweat_smile: